Cargo.toml

```diff
@@ -10,15 +10,17 @@ edition = "2021"
 # TODO better error types than anyhow
 anyhow = "1.0"
 ethers = { git = "https://github.com/gakonst/ethers-rs" }
+ethers-contract = "0.17.0"
```

Comment:
maybe we should have a monorepo and stick ipfs-proof-buddy in here too

Reply:
I feel like building core tools, and then implementations on top in separate repos is the way to go. Like maybe ipfs_proof_buddy could define both main and lib, and be importable that way, but why pack a whole Rocket server into a repo that's only gonna get bigger.

Reply:
That being said if there's anything generalizable in what u do have in proof_buddy, lets incorporate them.